### PR TITLE
adds Lycancoughy virus

### PR DESCRIPTION
**Pull request by Takov-Mala**
## About The Pull Request

Imports/converts the Lycancoughy virus from Paradise (which no longer uses it anyway), a disease that causes you to vomit doggos. Easily cured with ethanol.  

## Why It's Good For The Game

Corgis. 
Corgis everywhere.

## Changelog
:cl:Takov
add: Added Lycancoughy virus.
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
But why

**Comment by Time-Green** [comment]
fetish content

**Comment by Takov-Mala** [comment]
> 
> 
> fetish content

Corgos are not fetish content. 
And no, there wasn't any such thing in the original code either. (There was some stuff for spawning foxes which we lack and plushes we don't have, deleted all that.)

**Comment by PKPenguin321** [comment]
ah yes reverse vore

File: code/datums/diseases/lycan.dm [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+				to_chat(affected_mob, "<span class='danger'>Your stomach growls!</span>")
+		if(4)
+			if(prob(10))
+				to_chat(affected_mob, "<span class='danger'>Your stomach barks?!</span>")
+			if(prob(5))
+				affected_mob.visible_message("<span class='danger'>[affected_mob] awoos!</span>", \
+												"<span class='userdanger'>You awoo!</span>")
+				affected_mob.confused += (rand(6,8))
+			if(prob(3) && barklimit <= 10)
+				var/list/puppytype = list(/mob/living/simple_animal/pet/dog/corgi/puppy, /mob/living/simple_animal/pet/dog/pug, /mob/living/simple_animal/pet/dog/corgi/exoticcorgi, /mob/living/simple_animal/pet/dog/corgi/narsie, /mob/living/simple_animal/pet/dog/corgi/puppy/void )
```
Comment (ATH1909) [inline code/datums/diseases/lycan.dm]: Uh, you do know that Nars-Ian gibs other pets near him (with a 5% chance to do so each tick), right?

Comment (ATH1909) [reply code/datums/diseases/lycan.dm]:
```
/mob/living/simple_animal/pet/dog/corgi/narsie/Life()
	..()
	for(var/mob/living/simple_animal/pet/P in range(1, src))
		if(P != src && prob(5))
			visible_message("<span class='warning'>[src] devours [P]!</span>", \
			"<span class='cult big bold'>DELICIOUS SOULS</span>")
			playsound(src, 'sound/magic/demon_attack1.ogg', 75, TRUE)
			narsie_act()
			P.gib()
```

Comment (Takov-Mala) [reply code/datums/diseases/lycan.dm]: > 
> 
> Uh, you do know that Nars-Ian gibs other pets near him (with a 5% chance (per pet) to do so each tick), right?
> 
> The halls are going to be LITTERED with corgi gibs.

I did not.
I'm not sure if it should be changed tbh

**Comment by OverlyGeneric** [comment]
image

**Comment by Dawson1917** [comment]
hi hello can we have some useful viro symptoms instead of more fluff thanks god bless

**Comment by ATH1909** [comment]
> hi hello can we have some useful viro symptoms instead of more fluff thanks god bless

This isn't a virus symptom, you dum-dum. It's its own special disease, like Anxiety, Beesease, Wizarditis, Magnitis, etc.

**Comment by Dawson1917** [comment]
> 
> 
> > hi hello can we have some useful viro symptoms instead of more fluff thanks god bless
> 
> This isn't a virus symptom, you dum-dum. It's its own special disease, like Anxiety, Beesease, Wizarditis, Magnitis, etc.

i am d*m-dum

**Comment by actioninja** [comment]
A bit torn on this but going to go with "this is weird and thematically makes very little sense."

**Pull request closed without merging**